include/cutlass/gemm/warp/mma_mixed_input_tensor_op.h

```diff
@@ -492,6 +625,13 @@ class MmaMixedInputTensorOp {
     MmaOperandB const *ptr_B = reinterpret_cast<MmaOperandB const *>(&B);
     MmaOperandC *ptr_D = reinterpret_cast<MmaOperandC *>(&D);
 
+    if constexpr (elem_B_4bit) {
+      if (!even_flag_) {
+        ptr_B += TransformedFragmentB::kElements / 2 / MmaOperandB::kElements;
+      }
+      even_flag_ = !even_flag_;
+    }
+
```

Comment:
@alexsamardzic can you explain what are you trying to achieve here? and in general with a mutable `even_flag_` in this file.

Reply:
ok. I think I got it. Ideally, this should also be in iterator. You are loading double the amount of data for int4 operand.

Reply:
Yes, just because how iterators used are written currently, twice the amount of int4 data needed gets loaded on each `load()` call.  At the moment, as mentioned under 1. in my initial comment, there is a part of handling this in the testbed too, that apparently need to be moved elsewhere - so, open for suggestions for a better approach.